### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by amr-mokhtar**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (krishnajs) [inline CODE_OF_CONDUCT.md:1]: do we need to capture - if there is a conflict how would that be resolved ?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at info@mail.openness.org. All
```
Comment (krishnajs) [inline CODE_OF_CONDUCT.md:58]: is this mail archived and so if someone makes a complaint can others see it? Do we need to create unarchived/anonymous mail ?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (amr-mokhtar) [reply CODE_OF_CONDUCT.md:1]: Conflict resolution is handled by TSC. This is covered in the contribution guide.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (amr-mokhtar) [reply CODE_OF_CONDUCT.md:58]: Yes, indeed.. This is an archived inbox. I will update it to be contacted with OpenNESS maintainers.

**Pull request merged**